tests/eosio.system_tests.cpp

```diff
@@ -1381,8 +1433,6 @@ BOOST_FIXTURE_TEST_CASE(producer_pay, eosio_system_tester, * boost::unit_test::t
 
 BOOST_FIXTURE_TEST_CASE(multiple_producer_pay, eosio_system_tester, * boost::unit_test::tolerance(1e-10)) try {
 
-   auto within_one = [](int64_t a, int64_t b) -> bool { return std::abs( a - b ) <= 1; };
-
    const int64_t secs_per_year  = 52 * 7 * 24 * 3600;
    const double  usecs_per_year = secs_per_year * 1000000;
    const double  cont_rate      = 4.879/100.;
```

Comment:
Shouldn't this be:
```cpp
const double continuous_rate = std::log1p(double(0.05));
```

And a similar change for the one in `producer_pay`.

Furthermore, there are other changes in `producer_pay` dealing with `within_one` usage that were in #253, but are not in this PR.

Reply:
Never mind. I see now that these changes in #253 were regarding the inflation changes which does not apply to this PR.